src/pages/stylePage/ChooseAIStylePage.tsx

```diff
@@ -0,0 +1,205 @@
+
+import * as S from "./ChooseAIStylePage.styled";
+
+const ChooseAIStylePage: React.FC = () => {
+  // URL 목록은 한 번만 생성
+  const presetUrls = useMemo(
+    () => [
+      {
+        imgaeName: "sample1",
+        url: "https://images.unsplash.com/photo-1508214751196-bcfd4ca60f91?q=80&w=800&auto=format&fit=crop",
+      },
+      {
+        imgaeName: "sample2",
+        url: "https://images.unsplash.com/photo-1512496015851-a90fb38ba796?q=80&w=800&auto=format&fit=crop",
+      },
+      {
+        imgaeName: "sample3",
+        url: "https://images.unsplash.com/photo-1540206276207-3af25c08abc4?q=80&w=800&auto=format&fit=crop",
+      },
+    ],
+    []
+  );
+
+  const defaultItemInfo: ItemProps = useMemo(
+    () => ({ name: "", content: "", category: "" }),
+    []
+  );
+
+  // itemId 1~3: URL, 4: 업로드 타일
+  const initial: ContentsProps[] = useMemo(
+    () => [
+      { itemId: 1, itemImage: presetUrls[0].url, itemInfo: defaultItemInfo },
+      { itemId: 2, itemImage: presetUrls[1].url, itemInfo: defaultItemInfo },
+      { itemId: 3, itemImage: presetUrls[2].url, itemInfo: defaultItemInfo },
+      { itemId: 4, itemImage: undefined, itemInfo: defaultItemInfo },
+    ],
+    [presetUrls, defaultItemInfo]
+  );
+
+  const [contents, setContents] = useState<ContentsProps[]>(initial);
+
+  // 어떤 타일이 선택되었는지 (1~4 중 1개 또는 null)
+  const [selectedId, setSelectedId] = useState<number | null>(null);
+
+  // 4번 업로드 타일 프리뷰 관리
+  const [previewUrl, setPreviewUrl] = useState<string | null>(null);
+  const inputRef = useRef<HTMLInputElement | null>(null);
+
+  const openFile = () => inputRef.current?.click();
+
+  const handleFile = (file: File) => {
+    setContents((prev) =>
+      prev.map((c) => (c.itemId === 4 ? { ...c, itemImage: file } : c))
+    );
+    // 업로드 타일을 선택 상태로 전환
+    setSelectedId(4);
+  };
+
+  const removeFile = () => {
+    setContents((prev) =>
+      prev.map((c) => (c.itemId === 4 ? { ...c, itemImage: undefined } : c))
+    );
+    if (previewUrl) URL.revokeObjectURL(previewUrl);
+    setPreviewUrl(null);
+    if (inputRef.current) inputRef.current.value = "";
+    // 업로드 이미지를 지우면 선택도 해제
+    setSelectedId(null);
+  };
+
+  // 4번 타일 File/URL 변화에 따라 미리보기 URL 관리
+  useEffect(() => {
+    const item4 = contents.find((c) => c.itemId === 4);
+    const img = item4?.itemImage;
+
+    setPreviewUrl((prev) => {
+      if (prev && prev.startsWith("blob:")) URL.revokeObjectURL(prev);
+      return null;
+    });
+
+    if (img instanceof File) {
+      const url = URL.createObjectURL(img);
+      setPreviewUrl(url);
+      return () => URL.revokeObjectURL(url);
+    } else if (typeof img === "string") {
+      setPreviewUrl(img);
+    }
+  }, [contents]);
+
+  const uploaded4 = Boolean(contents.find((c) => c.itemId === 4)?.itemImage);
+
+  // 다음으로 버튼 활성화 조건:
+  // - selectedId가 1,2,3 중 하나면 OK
+  // - selectedId가 4면 업로드된 이미지가 있어야 OK
+  const canNext =
+    selectedId !== null && (selectedId !== 4 || (selectedId === 4 && uploaded4));
```

Comment:
P2) `selectId`가 `null`이 아니고, 4면 업로드 이미지가 있는 걸 의도한 것 같은데, 지금 `selectId`가 4인지를 검사하는 게 중복되는 로직인 것 같아서

```suggestion
  const canNext = selectedId && (selectedId !== 4 || uploaded4);
```

로 변경해 줘도 좋을 것 같아요!